packages/config/src/config.js

```diff
@@ -0,0 +1,47 @@
+import { pickBy } from 'ramda'
+import deepMerge from './utils/merge'
+
+let layers = []
+
+let activeConfiguration = {};
+
+function define(configuration) {
+  layers.push(configuration)
+}
+
+function init(profiles) {
+
+  // reverse the order of the profiles, least important handled first
+  // and overridden with values from the more important profiles
+  if (profiles) profiles.reverse()
+
```

Comment:
(nitpick) But the formatting here is strange. There is a whole block under that is to the right by a space, togheter with the if above seems like a complete block inside an if (and is actually not)